examples/tests/sleeptenmin.py

```diff
@@ -25,13 +25,12 @@ def test(self):
         length = int(self.params.get('sleep_length', default=600))
         method = self.params.get('sleep_method', default='builtin')
 
-        for cycle in xrange(0, cycles):
+        for _ in xrange(0, cycles):
             self.log.debug("Sleeping for %.2f seconds", length)
             if method == 'builtin':
                 time.sleep(length)
             elif method == 'shell':
                 os.system("sleep %s" % length)
-            self.report_state()
```

Comment:
Also OK. This reminds us that `report_state()` is one of those odd APIs, that are being "indirectly" used by `avocado.utils.process` and need a better solution.